modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/engine/CommerceOrderEngineImpl.java

```diff
@@ -225,6 +227,51 @@ public CommerceOrder call() throws Exception {
 			});
 	}
 
+	@Override
+	public CommerceOrder updateCommerceOrder(
+			String externalReferenceCode, long commerceOrderId,
+			long billingAddressId, long commerceShippingMethodId,
+			long shippingAddressId, String advanceStatus,
+			String commercePaymentMethodKey, String purchaseOrderNumber,
+			BigDecimal shippingAmount, String shippingOptionName,
+			BigDecimal shippingWithTaxAmount, BigDecimal subtotal,
+			BigDecimal subtotalWithTaxAmount, BigDecimal taxAmount,
+			BigDecimal total, BigDecimal totalDiscountAmount,
+			BigDecimal totalWithTaxAmount, CommerceContext commerceContext,
+			boolean recalculatePrice)
+		throws PortalException {
+
+		try {
+			return TransactionInvokerUtil.invoke(
```

Comment:
There already is a method called _executeInTransaction that you can use instead of defining the ransaction again.
Use that one, just to keep it aligned with the rest of the class.